Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -119,6 +134,8 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
 
     RefPtr<WebCore::GraphicsLayer> createGraphicsLayer(const String& name, GraphicsLayer::Type);
 
+    ElementsUnderMouse getElementsUnderMouse(const WebMouseEvent&) const;
```

Comment:
Write this as `interactablesForPluginPoint()` (choose your coordinate system) and provide the "for event" as a wrapper.

Reply:
Settled at `pdfElementTypesForPluginPoint()`. I've extracted the event position in plugin coordinates for now, but I'll make a wrapper to provide the "for event" bit when we're handling other events too.